### DISCUSSION TITLE
It does not appear in the Overview and not in Todays Dedections

### DISCUSSION THREAD

**Discussion by Bongo2** [mcguirepr89/BirdNET-Pi#1250] [General]
I put a USB microphone on for the test and played a eagle owl (Bubo bubo_Uhu) using of a mobile app.

It is also detected with 83%. But it does not appear in the Overview and not in Todays Dedections.
Is that because human or engine has been detected? But I was very quiet. Only that the fan of the PC has run.
Do I have to set anything else?
![grafik](https://github.com/user-attachments/assets/a44b286d-28e5-44ce-b698-11cc4f30f4d1)

**Comment by alexbelgium** [comment]
Very likely it is below your species threshold of detection. Try to use v1.0 of the model instead of v2.4. You should also move on to this repo which is more up to date https://github.com/Nachtzuster/BirdNET-Pi and for which I've opened a PR that should exactly solve your issue by saying why the detection is not saved https://github.com/Nachtzuster/BirdNET-Pi/pull/190

**Comment by Bongo2** [comment]
I migrated your fork and lowered the threshold. Now he also finds the Uhu.
